pkg/server/events.go

```diff
@@ -296,6 +299,14 @@ func (em *eventMonitor) handleEvent(any interface{}) error {
 
 // handleContainerExit handles TaskExit event for container.
 func handleContainerExit(ctx context.Context, e *eventtypes.TaskExit, cntr containerstore.Container) error {
+	nric, err := nri.New()
+	if err != nil {
+		log.G(ctx).WithError(err).Error("unable to create nri client")
+	}
+	nriSB := &nri.Sandbox{
+		ID: cntr.SandboxID,
+	}
+
```

Comment:
suggest a sandbox helper to do this block and the invoke

Reply:
What would a helper look like?